src/basic/terminal-util.c

```diff
@@ -1429,12 +1423,6 @@ int open_terminal_in_namespace(pid_t pid, const char *name, int mode) {
 
         pair[1] = safe_close(pair[1]);
 
-        r = wait_for_terminate_and_check("(sd-terminalns)", child, 0);
```

Comment:
Hrm, why is this dropped? The change isn't really straightforward to me, hence worth some explanations in commit msg?

Reply:
The idea was to use FORK_WAIT instead.